services/src/handlers/wms.rs

```diff
@@ -140,8 +142,10 @@ async fn get_map<C: Context>(
 
     let operator = workflow.operator.get_raster().context(error::Operator)?;
 
+    let thread_pool = ThreadPool::new(1); // TODO: use global thread pool
```

Comment:
I think it should be managed in the services-context together with the different dbs. the services-context could then have a method to create a new executioncontext instance